raiden/tests/scenarios/ci/long_path_mediated_transfers.yaml

```diff
@@ -0,0 +1,367 @@
+            - assert: {from: 10, to: 11, total_deposit: 1000, balance: 1000, state: "opened"}
+            - assert: {from: 11, to: 10, total_deposit: 1000, balance: 1000, state: "opened"}
+            - assert: {from: 11, to: 12, total_deposit: 1000, balance: 1000, state: "opened"}
+            - assert: {from: 12, to: 11, total_deposit: 1000, balance: 1000, state: "opened"}
+            - assert: {from: 12, to: 13, total_deposit: 1000, balance: 1000, state: "opened"}
+            - assert: {from: 13, to: 12, total_deposit: 1000, balance: 1000, state: "opened"}
+            - assert: {from: 13, to: 14, total_deposit: 1000, balance: 1000, state: "opened"}
+            - assert: {from: 14, to: 13, total_deposit: 1000, balance: 1000, state: "opened"}
+      - serial:
+          name: "Make one transfer from 0 to 1"
+          tasks:
+            - transfer: {from: 0, to: 1, amount: 1, expected_http_status: 200}
+      - wait: 10
+      - parallel:
+          name: "Assert after one transfer from 0 to 1"
+          tasks:
+            - assert: {from: 0, to: 1, total_deposit: 1000, balance: 999, state: "opened"}
+            - assert: {from: 1, to: 0, total_deposit: 1000, balance: 1001, state: "opened"}
+      - serial:
+          name: "Make one transfer from 1 to 0"
+          tasks:
+            - transfer: {from: 1, to: 0, amount: 1, expected_http_status: 200}
+      - wait: 10
+      - parallel:
+          name: "Assert after one transfer from 1 to 0"
+          tasks:
+            - assert: {from: 0, to: 1, total_deposit: 1000, balance: 1000, state: "opened"}
+            - assert: {from: 1, to: 0, total_deposit: 1000, balance: 1000, state: "opened"}
+      - serial:
+          name: "Make one transfer from 0 to 2"
+          tasks:
+            - transfer: {from: 0, to: 2, amount: 1, expected_http_status: 200}
+      - wait: 10
+      - parallel:
+          name: "Assert after one transfer from 0 to 2"
+          tasks:
+            - assert: {from: 0, to: 1, total_deposit: 1000, balance: 999, state: "opened"}
+            - assert: {from: 2, to: 1, total_deposit: 1000, balance: 1001, state: "opened"}
+      - serial:
+          name: "Make one transfer from 2 to 0"
+          tasks:
+            - transfer: {from: 2, to: 0, amount: 1, expected_http_status: 200}
+      - wait: 10
+      - parallel:
+          name: "Assert after one transfer from 2 to 0"
+          tasks:
+            - assert: {from: 0, to: 1, total_deposit: 1000, balance: 1000, state: "opened"}
+            - assert: {from: 2, to: 1, total_deposit: 1000, balance: 1000, state: "opened"}
+      - serial:
+          name: "Make one transfer from 0 to 3"
+          tasks:
+            - transfer: {from: 0, to: 3, amount: 1, expected_http_status: 200}
+      - wait: 10
+      - parallel:
+          name: "Assert after one transfer from 0 to 3"
+          tasks:
+            - assert: {from: 0, to: 1, total_deposit: 1000, balance: 999, state: "opened"}
```

Comment:
Why didn't you add the other asserts here, for channels like ``1<->2``

Reply:
I don't think that's necessary for this scenario. I think it's out of scope for it. I can add it if you really want, but I would just keep it as simple as possible.